gstcefsrc.cc

```diff
@@ -340,8 +375,17 @@ void BrowserClient::OnBeforeClose(CefRefPtr<CefBrowser> browser) {
   mElement->browser = nullptr;
   g_mutex_lock (&mElement->state_lock);
   mElement->started = FALSE;
-  g_cond_signal (&mElement->state_cond);
+  g_cond_broadcast (&mElement->state_cond);
```

Comment:
Why is this needed? Can there be multiple threads waiting on this condition variable?

Reply:
I don't really think so, as each browser belongs to a single pipeline/element, and all of them run in CEF's UI thread. (This was probably overkill from attempting to address @sdroege's concern about thread waiting.)

Reply:
ack, that's what I imagined, let's keep the `g_cond_signal` here then :)